src/CMake/FindANARI.cmake

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+# Project developers.  See the top-level LICENSE file for dates and other
+# details.  No copyright assignment is required to contribute to VisIt.
+
+#****************************************************************************
+# Modifications:
+#
+#*****************************************************************************
+
+#[=======================================================================[.rst:
```

Comment:
This looks like some useful `.rst` content embedded in the `.cmake` file. Cool! :sunglasses: Is there a way we can pull that into our rendered deverloper's docs? I mean, is that what this block is designed for?

Reply:
We have an internal tool that processes the `.rst` content and I have some default settings in my IDE that generates a lot of the skeleton code when creating new files of certain types. Internally we have a few CMake wizards that will convert the `.rst` content into internal documentation for most of our build scripts.

Reply:
Ok, so we could potentially just grab that `.rst` content using the `.. include::` directive (not the `.. literalinclude::` directive which takes it verbatim...we want the `.rst` we include to be *interpreted*).

That directive supports start and end line numbers but not start/end regexes. So, we could go to somewhere in our developer docs and do something like...

```
.. include:: ../../tools/dev/scripts/bv_support/bv_anari.sh
    :start-line: 11
    :end-line: 53
```

I dunno if that is super useful because the start and end line numberings may need to be updated as that verbiage changes. But, might be worth it?